packages/application-shell/src/components/navbar/navbar.tsx

```diff
@@ -414,9 +413,7 @@ const ApplicationMenu = (props: ApplicationMenuProps) => {
 
   return (
     <>
-      {props.menuType === 'application' && props.menu.shouldRenderDivider && (
```

Comment:
The `props.menuType === 'application'` is the actual problem, as `application` does not exist anymore.

Reply:
The rest of the changes are to simplify a bit the implementation and remove the need to pass `menuType` as prop.